apps/monarch/monarch-frontend/src/app/SearchTherapists.tsx

```diff
@@ -168,6 +180,12 @@ const EditButton: React.FC<{
     setEmail(therapist.email);
     setWebsite(therapist.website);
     setFamiliesHelped(0);
+    setDateJoined('2024-07-30');
```

Comment:
Let's also set `familiesHelped` and `dateJoined` to what the value is on the practitioner, instead of these defaults